cmake/Modules/FindComputeCpp.cmake

```diff
@@ -41,6 +41,9 @@ set(COMPUTECPP_BITCODE "spir64" CACHE STRING
   "Bitcode type to use as SYCL target in compute++")
 mark_as_advanced(COMPUTECPP_BITCODE)
 
+set(SYCL_LANGUAGE_VERSION "2017" CACHE STRING "SYCL version to use. Defaults to 1.2.1.")
+mark_as_advanced(SYCL_LANGUAGE_VERSION)
```

Comment:
It's probably fine to leave this as shown to users. Folk might want to change it. We might be too eager to mark some of these variables as advanced, probably my mistake. Does 2017 == 1.2.1? That's a little unclear to me

Reply:
`2017` is not an official value, just something we've decided to use since 1.2.1 came out in 2017. I expect this option to be mainly used to enable SYCL 2020 (and beyond) features, so the value for 1.2.1 shouldn't be problematic.

Will unmark it as advanced.